### ISSUE TITLE
number-only string interpreted as datetime in symbol layer.

### ISSUE THREAD

**Issue by IVData** [arnaudleclerc/AzureMapsControl.Components#47]
**Describe the bug**
When a symbol layer marker uses `TextOptions()` to define a property based label, the label is formatted as the year in a datetime.

**To Reproduce**
Create a symbol layer marker. Use this textField format:
```
string LabelFormat = @"[
    ""format"",
    [""get"", ""Label""],
    {
        ""text-font"": [""literal"", [""StandardFont-Bold""]],
        ""font-scale"": 1.2
    }
]";
```

and

```
TextField = new ExpressionOrString(
        System.Text.Json.JsonDocument.Parse(LabelFormat)
),
```

Any label with a whitespace-delimited section of numbers is interpreted as a DateTime. "1972" is a datetime. "Tx 1972" is a datetime, but "Tx1972" is not.

I have tried using the format `[""to-string"", [""get"", ""Label""]]` which didn't change behaviour.

**Expected behavior**
Don't format everything as a datetime.

**Screenshots**
![image](https://user-images.githubusercontent.com/895845/120572051-72b7e380-c46f-11eb-923b-8dc08e1415fb.png)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser: Edge v91

**Comment by arnaudleclerc** [comment]
Thanks for noticing that, I will try to take a look at it today

**Comment by arnaudleclerc** [comment]
There should be a version `1.7.1-beta0001` available in a couple of minutes, would you mind trying if this solves your issue ? 

All the inputs you gave me should now be displayed as a text instead of a datetime. If you want to display a datetime, you can still give a datetime format (like "2021-06-03") or a `DateTime` object on the properties of the feature.

Let me know if this works for you, if yes, I will publish a stable version.

**Comment by IVData** [comment]
Sorry to say, but I've just got home for the day and it's a long weekend here so I won't be back at work till Tuesday.

Mind if I test it then?

**Comment by arnaudleclerc** [comment]
Sure, there's no rush! Enjoy the weekend 😊

**Comment by IVData** [comment]
`1.7.1-beta0001` fixes the problem. Thanks very much :)

**Comment by arnaudleclerc** [comment]
Great, thanks for the feedback! I have published a stable 1.7.1 with the fix
